### ISSUE TITLE
Illegal property [TZUNTIL] in some included timezones

### ISSUE THREAD

**Issue by vgionco** [ical4j/ical4j#545]
**Describe the bug**
Some included timezone definitions contain the TZUNTIL property, and apparently that's not a standard property. 
This is a regression introduced in commit 56827ca77642643b0d1720ad92feaa250012bf82 and the two timezones affected, as far as I can see, are "Africa/Casablanca" and "Africa/El_Aaiun":

```
$ grep -r TZUNTIL src/main/resources/
src/main/resources/zoneinfo/Africa/Casablanca.ics:TZUNTIL:20870511T020001Z
src/main/resources/zoneinfo/Africa/El_Aaiun.ics:TZUNTIL:20870511T020001Z
src/main/resources/zoneinfo-global/Africa/Casablanca.ics:TZUNTIL:20870511T020001Z
src/main/resources/zoneinfo-global/Africa/El_Aaiun.ics:TZUNTIL:20870511T020001Z
```

**To Reproduce**
Compile the following Java class:
```
package cal;

import net.fortuna.ical4j.model.TimeZone;
import net.fortuna.ical4j.model.TimeZoneRegistry;
import net.fortuna.ical4j.model.TimeZoneRegistryFactory;

public class TzBug {
    public static void main(String[] args) {
        TimeZoneRegistry tzRegistry = TimeZoneRegistryFactory.getInstance().createRegistry();
        
        TimeZone tz1 = tzRegistry.getTimeZone("Europe/Rome");
        if (tz1 == null) {
            System.out.println("Timezone Europe/Rome not found.");
            return;
        }
        System.out.println(tz1.getVTimeZone().toString());
        
        TimeZone tz2 = tzRegistry.getTimeZone("Africa/Casablanca");
        if (tz2 == null) {
            System.out.println("Timezone Africa/Casablanca not found.");
            return;
        }
        System.out.println(tz2.getVTimeZone().toString());
    }
}
```
Running the class produces the following logs:
```
12:54:40.097 [main] WARN   net.fortuna.ical4j.model.TimeZoneRegistryImpl - Error occurred loading VTimeZone
net.fortuna.ical4j.data.ParserException: Error at line 6:Illegal property [TZUNTIL]
	at net.fortuna.ical4j.data.CalendarParserImpl.parse(CalendarParserImpl.java:162) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:183) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:171) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:158) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.model.TimeZoneLoader.loadVTimeZone(TimeZoneLoader.java:105) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.model.TimeZoneRegistryImpl.getTimeZone(TimeZoneRegistryImpl.java:186) [ical4j-3.0.22.jar:?]
	at cal.TzBug.main(TzBug.java:21) [classes/:?]
Caused by: java.lang.IllegalArgumentException: Illegal property [TZUNTIL]
	at net.fortuna.ical4j.model.PropertyBuilder.build(PropertyBuilder.java:61) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.DefaultContentHandler.endProperty(DefaultContentHandler.java:142) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$PropertyParser.parse(CalendarParserImpl.java:298) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$PropertyParser.access$1100(CalendarParserImpl.java:230) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$PropertyListParser.parse(CalendarParserImpl.java:216) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$ComponentParser.parse(CalendarParserImpl.java:435) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$ComponentParser.access$900(CalendarParserImpl.java:411) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl$PropertyListParser.parse(CalendarParserImpl.java:214) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl.parseCalendar(CalendarParserImpl.java:118) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl.parseCalendarList(CalendarParserImpl.java:184) ~[ical4j-3.0.22.jar:?]
	at net.fortuna.ical4j.data.CalendarParserImpl.parse(CalendarParserImpl.java:153) ~[ical4j-3.0.22.jar:?]
	... 6 more
Timezone Africa/Casablanca not found.
```

**Expected behavior**
The "Africa/Casablanca" timezone definition should be found and printed on console.

**Environment (please complete the following information):**
 - OS: Linux
 - Java 8

**Additional context**
Worked fine with 3.0.6; 3.0.22 and latest 3.0.29 are failing.

**Comment by benfortuna** [comment]
Thanks for the bug report. I've identified an additional specification that we haven't implemented, which includes the `TZUNTIL` property. 

I will add support for this shortly which should resolve this issue.

**Comment by benfortuna** [comment]
Fix included in release 3.1.3

**Comment by vgionco** [comment]
Thanks! 
confirmed working in 3.1.3
